monkey/monkey_island.py

```diff
@@ -1,3 +1,6 @@
+from gevent import monkey
+monkey.patch_all()
```

Comment:
Confusing name, wouldn't it be nicer to
from gevent import monkey as gevent_monkey

since our entire codebase uses monkey a lot